plugins/hdfsviewer/src/azkaban/viewer/hdfs/velocity/hdfs-proxy-js.vm

```diff
@@ -36,5 +37,11 @@ $(function() {
     };
     $.post("", requestData, successHandler);
   });
+
+  $('#proxyname').keyup(function(event) {
+    if (event.which == 13) {
```

Comment:
To improve readability, consider replacing the number with a constant?
e.g. ref:
See https://stackoverflow.com/questions/1465374/javascript-event-keycode-constants